src/Schema/Keywords/Minimum.php

```diff
@@ -39,9 +42,14 @@ class Minimum extends BaseKeyword
     public function validate($data, $minimum, bool $exclusiveMinimum = false): void
     {
         try {
-            Validator::numeric()->assert($data);
-            Validator::numeric()->assert($minimum);
-        } catch (ExceptionInterface $e) {
+            if (class_exists(NumericVal::class)) {
+                Validator::numericVal()->assert($data);
+                Validator::numericVal()->positive()->assert($minimum);
```

Comment:
This change added a new validation which doesn't allow **minimum** values to be equal, or lower than zero (see https://respect-validation.readthedocs.io/en/latest/rules/Positive/).
openapi specification seems to allow zero or negative values for both `minimum` and `maximum`, although not explicitly.
Was this intentional?

Reply:
I think it's a copypaste from `MultipleOf`. Would you mind to submit a PR? I think there are some other problems of the same kind

Reply:
Also maybe we need more tests around this, since this had pass unseen

Reply:
Sure! I spotted only **minimum** and **maximum** with this extra rule, I'll add any others if I find them.